packages/shared/src/modules/users/libs/types/user-get-all-item-response-dto.type.ts

```diff
@@ -1,6 +1,9 @@
 type UserGetAllItemResponseDto = {
-	email: string;
-	id: number;
+	user: {
```

Comment:
items: YourUserType[]

Reply:
Can you please explain why there should be items: YourUserType[]? Don't really understand why there should be an array .

Reply:
@what1s1ove this is an item type, so it should be like this